docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+services:
+  cinema_service:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db &&
+      python manage.py makemigrations &&

```

Comment:
Please avoid running `python manage.py makemigrations`. 
Migration files should already exist in the project by this stage